cli/command/manifest/fetch.go

```diff
@@ -44,23 +43,21 @@ func getLocalImageManifestData(namedRef reference.Named, transactionID string) (
 		return nil, nil, err
 	}
 
-	// If this is a manifest list, let's check for it locally so a user can see any modifications
-	// he/she has made.
+	// TODO: is this case relevant if transactionID == "" ?
```

Comment:
if transactionID is empty then you should be getting the whole list (e.g. someone did an inspect of a manifest list so there was no 2nd parameter)